components/ui/form/Schedule.tsx

```diff
@@ -62,6 +65,41 @@ const TimeRangeField = ({ name }: TimeRangeFieldProps) => {
     );
   }, []);
 
+  const [valueSelectStart, setValueStart] = useState<Option>(initialValueforSelectStart);
+
+  const customOnChange = (onChange: any, option: any) => {
+    onChange(new Date(option?.value as number));
+    setValueStart(option);
+  };
+
+  const calculateRange = (value: Option) => {
```

Comment:
@Jenietoc Could you please use more descriptive names in your code? Something like `calculateTimeRange`